### PR TITLE
Chore/new mapbox api

### PR DESCRIPTION
**Pull request by DanielHabenicht**

### CODE REVIEW

**Comment by kommander** [comment]
@DanielHabenicht can we move these settings to `apps/official/index.ts`? @weltenwort added a `beforeId` for the area layer, which is possibly a different one in the custom maps and needs to be adjusted (https://github.com/CovOpen/CovMapper/blob/master/apps/official/index.ts#L143)

File: src/components/GLMap.tsx [hunk 1]
```diff
@@ -71,12 +71,12 @@ export const GLMap = ({ mapRef, onMapClick, onViewportChange, onLoad }: GLMapPro
...
         maxZoom={MAX_ZOOM_LEVEL}
         minZoom={4}
-        mapStyle="mapbox://styles/mapbox/dark-v10"
         {...viewport}
         onClick={(evt) => onMapClick(evt, viewport)}
         onLoad={onLoad}
         onViewportChange={handleLocalViewportChange}
-        mapboxApiAccessToken="pk.eyJ1IjoiYWxleGFuZGVydGhpZW1lIiwiYSI6ImNrODFjNjV0NDBuenIza3J1ZXFsYnBxdHAifQ.8Xh_Y9eCFgEgQ-6mXsxZxQ"
+        baseApiUrl="https://openmap.covmap.aquiver.de"
+        mapStyle="https://openmap.covmap.aquiver.de/styles/osm-bright/style.json"
```
Comment (DanielHabenicht) [inline src/components/GLMap.tsx:79]:
```suggestion
        mapStyle="https://openmap.covmap.aquiver.de/styles/dark-matter/style.json"
```

**Comment by DanielHabenicht** [comment]
@weltenwort, how did you find out the matching layer?

**Comment by weltenwort** [comment]
I set a breakpoint in the debugger and inspected the mapbox layer stack. I suspect the layer list is also available in the style object returned by the [retrieve style API](https://docs.mapbox.com/api/maps/#retrieve-a-style).

**Comment by weltenwort** [comment]
Looking at https://github.com/openmaptiles/dark-matter-gl-style/blob/1282599bb8d58aad6ec6f42b6e0519c7612b566d/style.json#L729 I suspect it would have to be `place_other`.

**Comment by DanielHabenicht** [comment]
closed in favour of #148

**Pull request closed without merging**